webnn-status.md

```diff
@@ -335,6 +588,52 @@ The total number of WebNN ops is 78. This table currently lists ops that are imp
             dml = `<td></td><td>${s.dml_chromium_version_added}</td>`;
           }
 
+          let mlservice = '';
+          if (s.mlservice_op.toString().trim() === "") {
```

Comment:
My local test build stops on this line with an error:

`typeError: Cannot read properties of undefined (reading 'toString')`

And this is how `s` looks like just prior to the error:

```
{
    "op": "argMax",
    "op_id": "argminmax",
    "version": "",
    "wpt": "arg_min_max",
    "wpt_progress": 4,
    "xnnpack_op": [
        ""
    ],
    "xnnpack_progress": 2,
    "xnnpack_chromium_version_added": "",
    "dml_op": [
        "REDUCE_FUNCTION_ARGMAX"
    ],
    "dml_progress": 4,
    "dml_chromium_version_added": "M122",
    "tflite_op": [
        ""
    ],
    "tflite_progress": 2,
    "tflite_version_added": "",
    "ort_op": [
        "ArgMax"
    ],
    "ort_progress": 4,
    "ort_version_added": "main",
    "notes": ""
}
```
It's missing `mlservice_op`, thus error.

Can you double-check whether this is related to my local build or something you can reproduce too?